package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.5",
+    "@testing-library/user-event": "^12.7.1",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^12.20.1",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1",
```

Comment:
So we have those `@types/*` (typescript definitions) here on `dependencies`, do you think this is the right place for them?